src/Array/Extra.elm

```diff
@@ -767,9 +797,34 @@ Extra elements of either `Array` are glued to the end without anything in betwee
         |> interweave (repeat 1 "on")
     --> fromList [ "turtles", "on", "turtles", "turtles" ]
 
+@deprecated **Beware:** For historical reasons, this function takes it's arguments in the opposite order to `List.Extra.interweave`.
+As such, this function is deprecated in v1 and will be removed in v2. You should switch to `interweave_` which has the correct argument order. We plan to re-introduct `interweave` with the new argument order in a future release, but since the chance of subtle breakage is quite high, we will only do this gradually.
+
 -}
 interweave : Array a -> Array a -> Array a
 interweave toInterweave array =
+    interweave_ array toInterweave
+
+
+{-| Place all elements of a given `Array` between all current elements.
```

Comment:
With this argument order (non-pipe-friendly), the description seems off. Maybe change it to something like
> Place all elements of the second `Array` between all elements of the first.